data_processors/pipeline/lambdas/orchestrator.py

```diff
@@ -187,10 +187,10 @@ def next_step(this_workflow: Workflow, skip: dict, context=None):
             logger.info("Updating Google LIMS")
             google_lims_update_step.perform(this_workflow)
 
-        if "DRAGEN_WGS_QC_STEP" in skiplist:
-            logger.info("Skip performing DRAGEN_WGS_QC_STEP")
+        if "DRAGEN_WGTS_QC_STEP" in skiplist:
```

Comment:
Can you also update this new type in `handler()` event payload dict docstring, pls.

Reply:
Resolved in 1186883cb382396e469671b2dc8b2c768c39d448